TWLight/users/templates/users/my_applications.html

```diff
@@ -3,85 +3,101 @@
 {% load l10n %}
 
 {% block content %}
-  {% comment %}Translators: Heading of the applications page where users can view all of their applications listed with other relevant info.{% endcomment %}
-  <h3 class="inline-header">{% trans 'My applications' %}</h3>
-    <a href="{% url 'users:my_library' %}" class="btn btn-default pull-right">
-    {% comment %}Translators: A button on the 'your applications' page which when clicked takes users their collection page.{% endcomment %}
-    {% trans "My Library" %}
-  </a>
-  <div class="clearfix"></div>
-  {% for app in object_list %}
-    <div class="row">
-      <div class="col-sm-2 hidden-xs">
-        <span class="label label{{ app.get_bootstrap_class }} pull-right">
-          {% if app.parent %}
-            Renewal -
-          {% endif %}
-          {{ app.get_status_display }}
-        </span>
-      </div>
-      <div class="col-xs-8 col-sm-6">
-        <h4>
-        <a href="{{ app.get_absolute_url }}">
-          {{ app.partner }}
-          {% if app.specific_stream %}
-            ({{ app.specific_stream }})
-          {% endif %}
-        </a>
-        </h4>
-        <div class="visible-xs">
-          <span class="label label{{ app.get_bootstrap_class }}">
-            {% if app.parent %}
-              Renewal -
-            {% endif %}
-            {{ app.get_status_display }}
-          </span>
-          <br /><br />
-        </div>
-        {% if app.imported %}
-          {% comment %}Translators: On the page listing applications, this shows next to an application which was imported to the website.{% endcomment %}
-          {% trans 'Imported application' %}
-        {% elif app.get_version_count > 1 %} {# first version is original submission, not later review #}
-          {% if app.get_latest_reviewer %}
-            {% comment %}Translators: On the page listing applications, this shows next to an application which was previously reviewed. Don't translate {{ reviewer }} or {{ review_date }}.{% endcomment %}
-            {% blocktrans trimmed with reviewer=app.get_latest_reviewer review_date=app.get_latest_review_date|localize %}
-              Last reviewed by {{ reviewer }} on {{ review_date }}
-            {% endblocktrans %}
-          {% else %}
-            {% comment %}Translators: On the page listing applications, this shows next to an application which was previously reviewed. Don't translate {{ review_date }}.{% endcomment %}
-            {% blocktrans trimmed with review_date=app.get_latest_review_date|localize %}
-              Last reviewed on {{ review_date }}
-            {% endblocktrans %}
-          {% endif %}
-        {% else %}
-          {% comment %}Translators: On the page listing applications, this shows next to an application which has not yet been reviewed.{% endcomment %}
-          {% trans 'Not yet reviewed.' %}
+{% comment %}Translators: Heading of the applications page where users can view all of their applications listed with
```

Comment:
The indentation that was in this file before the last change you added was correct. Please change the indentation to the way it was before; it makes the code clearer and easier to read.